File: support_tools/swigwin-3.0.12/Examples/test-suite/go/varargs_runme.go
package main

import "./varargs"

func main() {
	if varargs.Test("Hello") != "Hello" {
		panic("Failed")
	}

	f := varargs.NewFoo("Greetings")
	if f.GetStr() != "Greetings" {
		panic("Failed")
	}

	if f.Test("Hello") != "Hello" {
		panic("Failed")
	}

	if varargs.Test_def("Hello", 1) != "Hello" {
		panic("Failed")
	}

	if varargs.Test_def("Hello") != "Hello" {
		panic("Failed")
	}
}

File: support_tools/swigwin-3.0.12/Examples/test-suite/ocaml/varargs_runme.ml
(* Test case stolen from the python directory *)

open Swig
open Varargs

let _ = if _test (C_string "Hello") <> (C_string "Hello") then
    raise (Failure "1")

let f = new_Foo C_void
let _ = if (invoke f) "test" (C_string "Hello") <> (C_string "Hello") then
    raise (Failure "2")
